### ISSUE TITLE
UI Issue with Existing OData Connected Services in Visual Studio 2022

### ISSUE THREAD

**Issue by nakozak** [OData/ODataConnectedService#309]
### Describe the bug
The Behavior of the ODataConnectedService Plugin after installing the latest release to VS2022 does not include the Update OData Connected Service... option in the menu dropdown for an existing set of OData Connected Services after a right-click. The Option does appear if you Add a new connected service to the project.

#### Version of the Project affected
ODataConnectedService v1.0.0
VisualStudio Version v17.2.6
Microsoft.Odata.Client v7.12.1

#### To Reproduce
1. Go to Solution
2. Right Click on Folder inside Connected Services (Existing Connected Services)
3. Update ODataConnected Service... option does not appear in Menu

#### Expected behavior
Right Click on Folder inside Connected Services
Update OData Connected Service... option should appear in Menu

#### Actual behavior
Right Click on Folder inside Connected Services
Update OData Connected Service... option doesn't appear in Menu for pre-existing Connected Services when using 2022. 

#### Screenshots
Visual Studio 2019 Menu Options after Right Click of Existing Connected Services
![VS2019](https://user-images.githubusercontent.com/21181117/182662929-fd359644-a366-47f7-9276-8a8f752ddaa2.png)

Visual Studio 2022 Menu Options after Right Click of Existing Connected Services
![VS2022](https://user-images.githubusercontent.com/21181117/182662948-b0771180-dcef-442d-b2bf-fbaba7f54967.png)
Extensions > Manage Extensions in VS2022
![VS2022 Extensions](https://user-images.githubusercontent.com/21181117/182663553-774a2b3c-0c36-40c7-a061-7bf28def4676.png)

#### Additional context
Side Request:
Can an option be added to disable the Dialog Window Output after hitting finish? This Plugin writes to the Output window and the OData Connected Service Dialog window, which can lead to an out of memory exception in non 64-bit versions of Visual Studio.

**Comment by ElizabethOkerio** [comment]
@nakozak this seems to be a Visual Studio issue. https://developercommunity.visualstudio.com/t/Connected-Services-plugins-cant-update-/10070285#T-ND10099077.

**Comment by nakozak** [comment]
Hi @ElizabethOkerio, I can confirm after Installing VS 17.3.0 Preview and Installing the ConnectedService Plugin, this UI issue is partially resolved. I'm able to achieve this successful result when opening Visual Studio, then opening the Solution (File > Open > Project/Solution), then waiting a bit for the Project and Extensions to be fully-loaded. 

The UI Issue still persists if opening a solution directly with the Preview Build, which doesn't seem to be open Visual Studio with Administrator Rights.

Steps to Reproduce:
1. Right-Click on Preview Visual Studio Build (if pinned to the TaskBar or from Start Window)
2. Select the Solution to Load
3. Perform same steps in the initial request to trigger the UI Issue

I noticed in Task Manager there are more processes under Visual Studio when opening the project from the Visual Studio menu and is being properly loaded as Admin

Working Option Task Manager View
![Open Project From VS Menu](https://user-images.githubusercontent.com/21181117/182910830-9f19f940-d811-4cdd-a80f-8f926657c538.png)

Non-Working Option Task Manager View
![Right-Click VS to Open Solution](https://user-images.githubusercontent.com/21181117/182910801-61f32108-92d0-48e7-a9d4-82daa1a64a65.png)

I was able to work around this by following these steps: https://stackoverflow.com/a/41758824 which allows for right-click open solution directly and the Extensions to Load Properly, resolving the UI Issue with Visual Studio for this Plugin.

Task Manager View After Applying Stack Overflow Suggestion:
![image](https://user-images.githubusercontent.com/21181117/182914395-37175bd3-d935-423b-ae0e-65c50cf696cf.png)



Hope this helps!
